scripts/fetchNodeSize.js

```diff
@@ -3,149 +3,106 @@ const axios = require("axios");
 const fs = require("fs");
 const path = require("path");
 
-// Function to calculate the daily increase using "Range", matching the "Reduce" transformation in Grafana
 const calculateDailyIncrease = (values) => {
-  // Sort values by timestamp to ensure they're in order
+  if (values.length < 2) return 0;
   values.sort((a, b) => a[0] - b[0]);
-
-  // Calculate the total time range in days
-  const timeRangeDays =
-    (values[values.length - 1][0] - values[0][0]) / (24 * 60 * 60);
-
-  // Find the minimum and maximum values
+  
+  const timeRangeDays = (values[values.length - 1][0] - values[0][0]) / (24 * 60 * 60);
   const minValue = Math.min(...values.map((v) => parseFloat(v[1])));
   const maxValue = Math.max(...values.map((v) => parseFloat(v[1])));
-
-  // Calculate the range (difference between max and min)
-  const rangeIncrease = maxValue - minValue;
-
-  // Convert to daily increase
-  const dailyIncrease = rangeIncrease / timeRangeDays;
-
-  return dailyIncrease;
+  return (maxValue - minValue) / timeRangeDays;
 };
 
-// Helper function to get week number
 function getWeekNumber(d) {
```

Comment:
We need to retain line 31 in this function otherwise there's a risk that week numbering will be wrong. The ISO system uses Thursday as an anchor day (hence the `+4`, where `0` is Monday, the week start) to ensure that the first week of the year is actually the first week of the year that has at least 4 days in the calendar year. See [here](https://en.wikipedia.org/wiki/ISO_week_date#First_week).